GameServer/Logic/Content/ItemManager.cs

```diff
@@ -125,6 +125,21 @@ private ItemType(Game game, ItemManager owner, ItemContentCollectionEntry itemIn
             GoldPerSecond = new StatModifcator();
         }
 
+        private int getTotalItemPrice()
```

Comment:
What was wrong with the `FindPrice()` function? You better fix that instead of re-implementing it.